CONTRIBUTING.md

```diff
@@ -67,3 +79,74 @@ If you want to take part of this please follow this process:
 * Create a commit with all changes.
 
 * Go to the next rubocop offense.
+
+# <a name="setup_backend"/>Setup an OBS backend for development
```

Comment:
No, it's not the same , e.g.  start_test_backend
* does not start workers
* runs bs_publish in testmode 
* etc.